src/BoundaryConditions/flux_conditions.jl

```diff
@@ -19,6 +19,16 @@ struct CustomFluxCondition <: AbstractBoundaryCondition
 end
 
 """
-    DragLawCondition <: AbstractFluxBoundaryCondition
+    DragLawCondition{C} <: AbstractFluxBoundaryCondition
 """
-struct DragLawCondition <: AbstractBoundaryCondition end
+mutable struct DragLawCondition{C} <: AbstractBoundaryCondition
+    coefficients::C
+end
+
+"""
+    BulkFormulaCondition{C, T}
+"""
+mutable struct BulkFormulaCondition{C, T} <: AbstractBoundaryCondition
```

Comment:
Just to note, `mutable` structs are not allowed on the GPU, so an instance of this struct cannot reside in a GPU kernel.